bitnami/mongodb/templates/standalone/dep-sts.yaml

```
@@ -434,6 +440,12 @@ spec:
             - key: mongodb-ca-key
               path: mongodb-ca-key
               mode: 0600
+        {{- else }}
+        - name: mongodb-certs-0
+          secret:
+            secretName: {{ .Values.tls.standalone.existingSecret }}
```

Same comment here about templates:

```suggestion
            secretName: {{ include "common.tplvalues.render" ( dict "value" .Values.tls.standalone.existingSecret "context" $) }}
```